gameMuster/views.py

```diff
@@ -42,11 +42,16 @@ def index(request):
     if 'rating' in data_from_filter:
         chosen_params['rating'] = int(data_from_filter['rating'])
 
-    game_list = games_manager.generate_list_of_games(genres=chosen_params['genres'],
-                                                     platforms=chosen_params['platforms'],
-                                                     rating=chosen_params['rating'])
+    game_list = list(set(Game.objects.filter(platforms__pk__in=chosen_params['platforms']
+                                                               or list(map(lambda x: x.pk,
+                                                                           Platform.objects.all())))
+                         .filter(genres__pk__in=chosen_params['genres']
+                                                or list(map(lambda x: x.pk,
+                                                            Genre.objects.all())))
+                         .filter(user_rating__gte=chosen_params['rating'])))
```

Comment:
Also you can use variables per filter instead of dict object `chosen_params`
```suggestion
    game_list = Game.objects
    if chosen_params['platforms']:
         game_list = game_list.filter(platforms__pk__in=chosen_params['platforms'])
    if chosen_params['genres']:
         game_list = game_list.filter(genres__pk__in=chosen_params['genres'])
    if chosen_params['rating']:
         game_list = game_list.filter(user_rating__gte=chosen_params['rating'])
    game_list = game_list.all()
```